test/cryptography/sign.js

```diff
@@ -116,6 +122,25 @@ ${defaultSecondSignature}
 			});
 	});
 
+	describe('#writeVarIt', () => {
+		it('should calculate correct value for n < 253', () => {
+			return expect(writeVarInt(5)).to.be.eql(Buffer.from([5]));
+		});
+		it('should calculate correct value for n < 65536', () => {
+			return expect(writeVarInt(300)).to.be.eql(Buffer.from([253, 44, 1]));
+		});
+		it('should calculate correct value for n > 65536', () => {
+			return expect(writeVarInt(65537)).to.be.eql(Buffer.from([254, 1, 0, 1, 0]));
+		});
```

Comment:
We should have test cases for `0`, a negative number, a number out of range (too large), a float, `undefined`(?), a buffer (should it handle buffers?)

Reply:
I'm ok with `0`, but don't agree with negative, `float` and `undefined` because of this function accepts only length of messages which passed from the parent function. Or you see other cases? I think this is overtesting.

Reply:
Fair enough.